pubtools/_quay/iib_operations.py

```diff
@@ -127,8 +127,11 @@ def task_iib_add_bundles(
     )
 
     # Sign image
+    index_stamp = timestamp()
```

Comment:
I wonder, is it possible that two images may end up with the same tag?

Reply:
What about two parallel pushes?

Reply:
I think there's almost 0 chance it can happen.